tests/integration/resource_manager/V2/test_engine.py

```diff
@@ -139,3 +139,31 @@ def test_engine_update_multiple_parameters(
     finally:
         engine.stop()
         engine.delete()
+
+
+def test_engine_rename(
+    auth: Auth,
+    account_name: str,
+    api_endpoint: str,
+    database_name: str,
+    engine_name: str,
+):
+    rm = ResourceManager(
+        auth=auth, account_name=account_name, api_endpoint=api_endpoint
+    )
+    name = f"{engine_name}_to_rename"
+    new_name = f"{engine_name}_renamed"
+    engine = rm.engines.create(name=name)
```

Comment:
Ideally this would not start engine and create a stopped one so the whole test is faster and easier on the backend. However, looks like we don't expose autostart parameter, right?

Reply:
Correct, we don't expose that yet